src/psyclone/psyGen.py

```diff
@@ -5284,11 +5307,62 @@ def local_symbols(self):
         :return:  List of local symbols.
         :rtype: list of :py:class:`psyclone.psyGen.Symbol`
         '''
-        locals_list = []
-        for symbol in self._symbols.values():
-            if symbol.scope == "local":
-                locals_list.append(symbol)
-        return locals_list
+        return [sym for sym in self._symbols.values() if sym.scope == "local"]
+
+    @staticmethod
+    def indent(count, indent=INDENTATION_STRING):
```

Comment:
It seems wrong that we have a second `indent()` method. Can we rationalise and have just one?

Reply:
replaced by using Node.indent()

Reply:
Great.